Packs/XSOARSummaryDashboard/pack_metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "XSOAR Summary Dashboard",
+    "description": "Dashboard that shows overall platform performance as well as support links and cheat sheets for reference. The dashboard also pulls the most recent XSOAR live community blog posts.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "emitchell",
+    "url": "",
+    "email": "",
+    "created": "2023-01-11T17:47:48Z",
+    "categories": [],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+    "marketplaces": [
+        "xsoar",
+        "marketplacev2"
+    ],
```

Comment:
If it's only for XSOAR (and not for XSIAM) then let's remove the `marketplacev2` here

Reply:
That works for me, I contributed this through the XSOAR gui so I am guessing if you contribute content through there is automatically sets the marketplaces to xsoar and marketplacev2?